### PR TITLE
Add D110 Default Input Dataset

### PR DESCRIPTION
**Pull request by AdrianoDee**
This PR proposes the addition of a default dataset for D110 geometry. To be used, also, for the standard MinBias input and the inputs for the bot tests (to be introduced after this PR).

### CODE REVIEW

**Comment by AdrianoDee** [comment]
test parameters:
- workflows=29634.0,29834.0,29834.402

**Comment by AdrianoDee** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -4187,6 +4187,7 @@ def gen2023HiMix(fragment,howMuch):
 defaultDataSets['2026D95']='CMSSW_13_1_0_pre1-130X_mcRun4_realistic_v2_2026D95noPU-v'
 defaultDataSets['2026D96']='CMSSW_13_1_0_pre1-130X_mcRun4_realistic_v2_2026D96noPU-v'
 defaultDataSets['2026D98']='CMSSW_13_2_0_pre1-131X_mcRun4_realistic_v5_2026D98noPU-v'
+defaultDataSets['2026D110']='CMSSW_14_1_0_pre2-140X_mcRun4_realistic_v3_STD_2026D110_noPU-v'
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:4190]: Would you mind to add the comment on GT here, so we don't question in future if GT is wrong.

Comment (AdrianoDee) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: Sure. Maybe I was thinking to directly have a new production with the proper GT string.

**Comment by AdrianoDee** [comment]
test parameters:
- workflows=29634.0,29834.0,29834.402
- workflow_opts = -w upgrade

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
hold
- will updated with a new dataset with a new GT string in production (GS only)

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
test parameters:
- workflows=29634.0,29834.0
- workflow_opts = -w upgrade

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
unhold

**Comment by AdrianoDee** [comment]
+pdmv 
(workflows with D110 inputs successful)

**Comment by srimanob** [comment]
+Upgrade

**Comment by rappoccio** [comment]
+1

**Pull request merged**